test/tests/neml_damage/tests

```diff
@@ -0,0 +1,13 @@
+[Tests]
+  issues = '#305'
+  design = 'NEMLStress.md'
+  [mazars_damage]
+    requirement = 'BlackBear shall output the damage value for NEML models.'
+    [neml_damage_output]
```

Comment:
The SQA check is failing because you're missing the 'detail' parameter in the inner nested block. You really don't need the two levels of nesting here, and if you just flatten out everything that's in the [mazars_damage] block into one level, you don't need that parameter.